tools/vlsvdiff.cpp

```diff
@@ -1759,6 +1778,10 @@ int main(int argn,char* args[]) {
 
    // 4th arg is its component, 0 for scalars, 2 for z component etc
    uint compToExtract = atoi(argsVector[4].c_str());
+   if (compToExtract > 2){
```

Comment:
Incorrect, we do have some variables which can have more components. Not many yet, but the precipitating flux datareducer is one.